src/frontend/assets/VeilederSnakkeboble.tsx

```diff
@@ -10,6 +10,8 @@ const VeilederSnakkeboble: React.FC<VeilederProps> = props => (
             xmlns="http://www.w3.org/2000/svg"
             viewBox="0 0 54 93"
             aria-labelledby="veileder-title"
+            focusable={false}
+            role={'img'}
```

Comment:
Trenger man aria-label hvis det blir gjort om til img? eller trenger man da en alt-tekst i stedet for?

Reply:
Arc foreslår fortsatt aria-labelledby selv om role er img ja

Reply:
Oki 👍

Reply:
Står også her at man burde bruke labelledby https://a11y-guidelines.orange.com/en/articles/accessible-svg/